Source/Provers/SMTLib/SMTLibInteractiveTheoremProver.cs

```diff
@@ -647,6 +653,7 @@ protected override void Send(string s, bool isCommon)
 
       if (Process != null)
       {
+        SentSize += s.Length;
```

Comment:
It would be lovely if this could only be done from a wrapper around `Send`, to ensure that it's always incremented. It would be even nicer if the reset to size 0 could be encapsulated somewhere, too, though I remember from working on this code that there isn't a clear place to put it at the moment.